### PR TITLE
Fhir 36251 externalcodesystems

### PR DESCRIPTION
**Pull request by bmilius-nmdp**
Edited codings.html per FHIR-36251

### CODE REVIEW

**Review by sirkyp — APPROVED** [review]
Looks good to me!

**Review by James-R-Jones — APPROVED** [review]
Looks good

**Review by jschneid-nmdp — CHANGES_REQUESTED** [review]
Comment posted to Zulip:
> HL7's intention is to eventually have [terminology.hl7.org](http://terminology.hl7.org/) (a.k.a. THO) become the authoritative source for code system information.
> 
> The THO External Terminologies page already lists several genomics-related code systems, such as GTR, Hgnc, Hgvs, and ISCN.
> https://terminology.hl7.org/external_terminologies.html
> 
> It would be best to prominently mention the [terminology.hl7.org](http://terminology.hl7.org/) site on the CG IG's codings page.

Additional nitpicks:
* The HTA home page on Confluence is https://confluence.hl7.org/display/TA (not ~/External+Terminologies+-+Information).  Additionally, the HTA is now a subgroup of the TSMG:  https://confluence.hl7.org/display/TSMG
* This Confluence page describes a process for due diligence when searching for code system information, checking sources such as THO, the FHIR spec, the OID registry, etc.:  https://confluence.hl7.org/display/TA/External+Code+System+Owner+Engagement+Process

I'm thinking we should do some additional wordsmithing on the header paragraph for the codings page.

**Comment by jschneid-nmdp** [comment]
Another thought - for those code systems with a THO page, it would be best to reference THO rather than Confluence.

**Review by patrick-werner — CHANGES_REQUESTED** [review]

**Review by patrick-werner — APPROVED** [review]

File: input/pages/codings.xml [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
+    <p>For those organizations that list the URI as "not assigned yet," check the <a href="https://terminology.hl7.org/external_terminologies.html">THO</a> or <a href="https://confluence.hl7.org/display/TA/External+Terminologies+-+Information">HTA sites</a> to see if it has been added since the publication of this IG. If not, we recommend following the <a href="https://confluence.hl7.org/display/TA/External+Code+System+Owner+Engagement+Process">External Code System Owner Engagement Processing</a>. As a last resort, we suggest using the organization site URL, change any "https" to "http", and remove trailing slashes.</p>
+    <p>Please contact the <a href="http://www.hl7.org/Special/committees/clingenomics/leadership.cfm">Clinical Genomics Workgroup</a> co-chairs if you have any questions about using these genomics code systems.</p>
```
Comment (sirkyp) [inline input/pages/codings.xml:16]: Should we use this link instead?
https://confluence.hl7.org/display/CGW

Comment (bmilius-nmdp) [reply input/pages/codings.xml:16]: I considered that, but the other site actually lists email addresses and a link to send email to all co-chairs. btw, I couldn't find where the vocab facilitator is listed. But I don't have a problem with linking to confluence.

Comment (sirkyp) [reply input/pages/codings.xml:16]: I would prefer linking to confluence, and if needed, we improve the confluence page.

Comment (bmilius-nmdp) [reply input/pages/codings.xml]: done

**Review by bheale — COMMENTED** [review]
ok

**Pull request merged**